Arduino/Propulsion/src/PropulsionModule.cpp

```diff
@@ -36,7 +41,18 @@ uint16_t byteToShort(int16_t index);
 void errorReply(uint8_t errorByte);
 
 /*****Propulsion Module Specific Parameters**************************/
+#ifdef TINYDUINO
+MotorDriver motor(0);
+void initMotor() {
+  motor.begin(127);
```

Comment:
Does the motor speed range from 0-255 with 127 being zero speed?

Reply:
I'll double-check this—127 here should indicate the maximum PWM value

Reply:
It should also be a constant, so I'll do that

Reply:
From what I understand, this is indeed what value should represent max forward speed, so 127 is correct